Doc/reference/datamodel.rst

```diff
@@ -2590,6 +2591,19 @@ through the object's keys; for sequences, it should iterate through the values.
    the container.
 
 
+.. method:: object.keys(self)
+
+   This method can be used by Python in arbitrary circumstances to treat an
+   arbitrary instance as a mapping or not.  The method should return an object
+   that can iterate over all the keys of the mapping.
+
+   .. impl-detail::
+
+      In CPython, the method is used by the ``**expression`` syntax in
+      :ref:`function calls <calls>`, as well as in the implementation of the
+      :meth:`dict.update` method of the native dictionnary class.
```

Comment:
```suggestion
      :meth:`dict.update` method of the native dictionary class.
```

just a small thing

Reply:
Why not even:
```suggestion
      :meth:`~dict.update` method of the native :class:`dict` class.
```

Reply:
That could work too, what does the ~ do, it removes the "dict." from the output ?

Reply:
yes!

Reply:
done